### PR TITLE
Basic settings without header

### PR DESCRIPTION
**Pull request by laurent-mbuyu**
I have created the settings UI. The header task was assigned to Rendy.

Let me know of anything that need attention and modification.

Thanks :)

### CODE REVIEW

File: src/components/settings.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+class Settings extends Component {
+
+    render(){
+
+        return (
+
+            <div>
+
+                <img src={'http://www.the-lizard.org/images/phocagallery/Mammals/Rabbit_320_Natural%20England_Allan%20Drewitt.jpg'} alt="profile picture" className="profileImage" />
```
Comment (RetroOosthuizen) [inline src/components/settings.js]: Could you please download the jpg file and save it locally under img. Thanks :)

**Review by RetroOosthuizen — CHANGES_REQUESTED** [review]

File: src/components/settings.js (hunk 1 above)
Comment (jaderabbit) [reply src/components/settings.js]: :+1:

File: src/index.js [hunk 2]
```diff
@@ -4,5 +4,10 @@ import './index.css';
 import App from './App';
 import registerServiceWorker from './registerServiceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+// Insert custom made components
+
+import Settings from './components/settings';
```
Comment (KimberleighBeyl) [inline src/index.js]: Why is settings the main page?

**Review by KimberleighBeyl — APPROVED** [review]
LGTM :+1:

**Review by RetroOosthuizen — APPROVED** [review]

**Pull request merged**